README.md

```diff
@@ -22,6 +22,7 @@ The following environment variables are supported (incl. example values):
 - `DIR_DATE_PATTERN:` "%Y%m%d"
 - `FULL_BACKUP_DATE_FORMAT:` "%a"
 - `FULL_BACKUP_DATE_RESULT:` "Sun"
+- `BEFORE_BACKUP_SCRIPT:` "/backup/before_script.sh"
```

Comment:
could you please add the information about the first parameter that you hand over to the before script.